iphone/Maps/UI/PlacePage/Components/PlacePageInfoViewController.swift

```diff
@@ -101,6 +101,8 @@ class PlacePageInfoViewController: UIViewController {
   private var wheelchairView: InfoItemViewController?
   private var driveThroughView: InfoItemViewController?
 
+  private var viewDidAppear = false
```

Comment:
This flag looks misleading, as a view may appear/disappear several times. Are there ways to avoid using a boolean at all?

Reply:
As VC may appear several times per lifetime and we should skip only the 1st appearing the bool cannot be skipped.
The only question is where it should be placed?
 
Interactor is ok too. But view didAppear is safer, because it is called later in comparison with viewWillApperar